src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/OrderPreferences/GeneralType.php

```diff
@@ -73,23 +73,57 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $currencyIsoCode = $this->defaultCurrencyIsoCode;
 
         $builder
-            ->add('enable_final_summary', SwitchType::class)
-            ->add('enable_guest_checkout', SwitchType::class)
-            ->add('disable_reordering_option', SwitchType::class)
+            ->add('enable_final_summary', SwitchType::class, [
+                'required' => false,
+                'label' => $this->trans('Enable final summary', 'Admin.Shopparameters.Feature'),
+                'help' => $this->trans('Display an overview of the addresses, shipping method and cart just before the order button (required in some European countries).', 'Admin.Shopparameters.Help'),
+            ])
+            ->add('enable_guest_checkout', SwitchType::class, [
+                'required' => false,
+                'label' => $this->trans('Enable guest checkout', 'Admin.Shopparameters.Feature'),
+                'help' => $this->trans('Allow guest visitors to place an order without registering.', 'Admin.Shopparameters.Help'),
+            ])
+            ->add('disable_reordering_option', SwitchType::class, [
+                'required' => false,
+                'label' => $this->trans('Disable Reordering Option', 'Admin.Shopparameters.Feature'),
+                'help' => $this->trans('Disable the option to allow customers to reorder in one click from the order history page (required in some European countries).', 'Admin.Shopparameters.Help'),
+            ])
             ->add('purchase_minimum_value', MoneyWithSuffixType::class, [
+                'required' => false,
+                'label' => $this->trans('Minimum purchase total required in order to validate the order', 'Admin.Shopparameters.Feature'),
+                'help' => $this->trans('Set to 0 to disable this feature.', 'Admin.Shopparameters.Help'),
                 'currency' => $currencyIsoCode,
                 'suffix' => $this->trans('(tax excl.)', 'Admin.Global'),
             ])
-            ->add('recalculate_shipping_cost', SwitchType::class);
+            ->add('recalculate_shipping_cost', SwitchType::class, [
+                'required' => false,
+                'label' => $this->trans('Recalculate shipping costs after editing the order', 'Admin.Shopparameters.Feature'),
+                'help' => $this->trans('Automatically updates the shipping costs when you edit an order.', 'Admin.Shopparameters.Help'),
+            ]);
 
         if ($isMultishippingEnabled) {
-            $builder->add('allow_multishipping', SwitchType::class);
+            $builder->add('allow_multishipping', SwitchType::class, [
+                'required' => false,
+                'label' => $this->trans('Allow multishipping', 'Admin.Shopparameters.Feature'),
+                'help' => $this->trans('Allow the customer to ship orders to multiple addresses. This option will convert the customer\'s cart into one or more orders .', 'Admin.Shopparameters.Help'),
```

Comment:
```suggestion
                'help' => $this->trans('Allow the customer to ship orders to multiple addresses. This option will convert the customer\'s cart into one or more orders.', 'Admin.Shopparameters.Help'),
```

Reply:
Can you confirm the "/" sign will not be taken into account?

Reply:
I confirm